lib/Linux/Statm/Tiny.pm

```diff
@@ -140,25 +135,25 @@ foreach my $attr (keys %stats) {
 
     has $attr => (
         is       => 'lazy',
-        isa      => Int,
+        isa      => 'Int',
         default  => sub { shift->statm->[$stats{$attr}] },
         init_arg => undef,
-        alias    => \@aliases,
         clearer  => "_refresh_${attr}",
+        @aliases ? (alias => \@aliases) : (),
```

Comment:
Does Mite not cope with an empty array of aliases? The original approach seems simpler imo.